Comparing /home/adrian/.m2/repository/org/apache/flink/flink-java/1.1.4/flink-java-1.1.4.jar with /home/adrian/Masterarbeit/Code/flink/flink-java/target/flink-java-1.3.0.jar:
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.java.ExecutionEnvironment  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) int getDefaultLocalParallelism()
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.java.operators.Operator  (not serializable)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.operators.ResourceSpec getMinResources()
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.api.common.operators.ResourceSpec getPreferredResources()
